### ISSUE TITLE
Drop the weird popover window (the one with the X) when you click in the CoffeeScript area

### ISSUE THREAD

**Issue by lipis** [js2coffee/js2coffee#363]
This is very annoying, because how it used to work (and it was very pleasant) is that you could just switch from one to another and continue writing.. now for some reason if you just accidentally click on in the CoffeeScript area you have to close the overlay to continue writing..

Does that makes any sense?

**Comment by bennyn** [comment]
I was also wondering what this is. The previous **js2coffee.org** site was much more easier to use. I like the URL change to **js2.coffee** but this pop over (with the X) is very annoying as you cannot just click in it and start to type (which was possible in the previous design of the website).

**Comment by pronebird** [comment]
I agree, it's more like preview of coffee to js but since translation is not literal I guess the idea was to avoid fixing original js hence the pop up was introduced.

**Comment by lipis** [comment]
Yes... but many times you go there, to quickly do something, possibly edit it and then copy back.. now I have to copy it first.. on missclicks is doing something else.. :)

**Comment by manuel-di-iorio** [comment]
+1
